testing/smoke/supported-os/test.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -eo pipefail
+
+KEY_NAME="provisioner_key"
+
+ssh-keygen -f ${KEY_NAME} -N ""
+
+# Load common lib
+. $(git rev-parse --show-toplevel)/testing/smoke/lib.sh
+
+# Get all the versions from the current region.
+get_latest_snapshot
+
+VERSION=${1}
+if [[ -z ${VERSION} ]] || [[ "${VERSION}" == "latest" ]]; then
+    VERSION=$(echo ${VERSIONS} | jq -r 'last')
+    echo "-> unspecified version, using $(echo ${VERSION} | cut -d '.' -f1-2)"
+fi
+MAJOR_VERSION=$(echo ${VERSION} | cut -d '.' -f1 )
+MINOR_VERSION=$(echo ${VERSION} | cut -d '.' -f2 )
+
+OBSERVER_VERSION=$(echo ${VERSION} | cut -d '-' -f1 )
+
+if [[ ${MAJOR_VERSION} -eq 7 ]]; then
```

Comment:
Are we testing old versions of APM Server or the stack? Is this code path relevant?

Reply:
I don't think we are. Good catch! :+1: